Cargo.toml

```diff
@@ -23,39 +23,46 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["hyper-server"]
-hyper-server = ["http-service-hyper"]
+default = ["h1-server"]
+hyper-server = ["futures", "hyper", "http", "tokio"]
+h1-server = ["http-service-h1"]
 docs = ["unstable"]
 unstable = []
 
 [dependencies]
-futures = "0.3.1"
-http = "0.1.19"
-http-service = "0.4.0"
-http-service-hyper = { version = "0.4.1", optional = true }
+http-types = "1.0.1"
+http-service = { git = "https://github.com/dignifiedquire/http-service-1", branch = "h1" }
+http-service-h1 = { git = "https://github.com/dignifiedquire/http-service-1", branch = "h1", optional = true }
```

Comment:
Why both of these is the same git repository?

Reply:
because they are both in the same repository?